packages/ui/dialogs/settlement/index.tsx

```diff
@@ -112,6 +118,8 @@ class Settlement extends Component<Props, State> {
     const denomination = 'ETH'
     let success
 
+    const settleTotal = Math.abs(balance) === Number(amount.replace(/[^0-9]/g, ''))
+    console.log('DIRECTION', direction, 'AMOUNT', amount, typeof amount)
```

Comment:
Can we make these console.log() calls conditional on Debug, e.g. is there a React equivalent of this?
#if DEBUG
console.log("blah")
#endif